### ISSUE TITLE
Camelot returns "bytes or integer address expected instead of str instance"

### ISSUE THREAD

**Issue by PDM996** [camelot-dev/camelot#359]
Hi,

I am running an Anaconda installation, and both dependencies are correctly installed, according to testing procedure.
When I run `camelot.read_pdf('table.pdf')` the prompt returns the error below.

_bytes or integer address expected instead of str instance_

Do you have any idea what the problem is and how I could fix it?

I have tried on both Jupyter Lab and Spyder.

Cheers,
Pietro
[table.pdf](https://github.com/camelot-dev/camelot/files/11018953/table.pdf)

**Comment by saidakyuz** [comment]
It works for me fine on Colab. Did you try full path as argument?

**Comment by PDM996** [comment]
I have tried multiple ways, even moving the _"table.pdf"_ file in a different folder without white spaces. I keep getting the same TypeError.
Below is a screenshot of the error.
![image](https://user-images.githubusercontent.com/123757660/226573410-74385dbb-bc40-4c20-8d36-62b6184fa7c9.png)

Might it be that I did not install it correctly?
As I mentioned before, when running the tests for both GhostScript and Tkinter, they are successful...and I have tried to reinstall Camelot multiple times in case something when wrong.
